### PR TITLE
Add paywall to be shown below article content

### PR DESCRIPTION
**Pull request by GeriYatola**
https://kalevamedia.atlassian.net/browse/L2-1771

### CODE REVIEW

File: components/wall_with_content.html [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+            });
+        }, 0);
+    }
+
+    if (document.querySelector('.paywall-css-container .paywall__main-title') || document.readyState === 'complete') {
+        init();
+    } else {
+        window.addEventListener('load', init);
+    }
+</script>
```
[the remark this answers is not in the document]
Comment (GeriYatola) [reply components/wall_with_content.html]: Fixed in 04bed20

File: README.md [hunk 2]
```diff
@@ -2,8 +2,36 @@
...
+## Test environment
+
+In test environment, components are served from S3 endpoint
+https://zephr-components-ci.s3.eu-west-1.amazonaws.com/
+
+Deploy component to test environment by running `bin/deploy.sh lorien-test`
+
+## Production
+
+In test environment, components are served from S3 endpoint
```
Comment (pietariheinovincit) [inline README.md:25]: "In prod environment" maybe?

Comment (GeriYatola) [reply README.md]: Thanks, missed this after copy&paster. Fixed

**Review by pietariheinovincit — APPROVED** [review]
Nice :+1:

**Pull request merged**